src/Raven.Pal/README.txt

```diff
@@ -1,10 +1,13 @@
 How to build PAL
+Prerequisites wsl of ubuntu 20.04 or lower
```

Comment:
Explain that we want to keep compatability here with older versions of GLIBC and we require GLIBC 2.3 or lower